providers/aws/resources/aws_s3.go

```diff
@@ -164,13 +164,25 @@ func (a *mqlAwsS3Bucket) id() (string, error) {
 	return a.Arn.Data, nil
 }
 
-func (a *mqlAwsS3Bucket) policy() (*mqlAwsS3BucketPolicy, error) {
-	bucketname := a.Name.Data
-
-	location := a.Location.Data
+func emptyAwsS3BucketPolicy(runtime *plugin.Runtime) (*mqlAwsS3BucketPolicy, error) {
+	res, err := CreateResource(runtime, "aws.s3.bucket.policy", map[string]*llx.RawData{
+		"name":       llx.StringData(""),
+		"document":   llx.StringData("{}"),
```

Comment:
without this, it's impossible to parse the document, since json-unmarshal expect it to at least read the structure. Without the document, we cannot determine an ID for the object, which is mandatory.

Reply:
oooh! thank you!